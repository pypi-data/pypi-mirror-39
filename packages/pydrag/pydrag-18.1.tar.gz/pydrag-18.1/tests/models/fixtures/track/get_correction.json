{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&artist=AC+%2F+DC&format=json&method=track.getCorrection&track=Hells+Bell"
            },
            "response": {
                "body": {
                    "string": "{\"corrections\":{\"correction\":{\"track\":{\"name\":\"Hells Bells\",\"mbid\":\"CENSOREDb286\",\"url\":\"https://www.last.fm/music/AC%2FDC/_/Hells+Bells\",\"artist\":{\"name\":\"AC/DC\",\"mbid\":\"CENSORED3ed1\",\"url\":\"https://www.last.fm/music/AC%2FDC\"}},\"@attr\":{\"index\":\"0\",\"artistcorrected\":\"1\",\"trackcorrected\":\"1\"}}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "344"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 16:15:30 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
