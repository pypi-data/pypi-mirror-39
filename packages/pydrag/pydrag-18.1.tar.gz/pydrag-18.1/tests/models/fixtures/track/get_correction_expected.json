{
    "artist": {
        "mbid": "CENSORED3ed1",
        "name": "AC/DC",
        "url": "https://www.last.fm/music/AC%2FDC"
    },
    "mbid": "CENSOREDb286",
    "name": "Hells Bells",
    "url": "https://www.last.fm/music/AC%2FDC/_/Hells+Bells"
}
